src/tls13.c

```diff
@@ -3540,7 +3547,10 @@ static int DoPreSharedKeys(WOLFSSL* ssl, const byte* input, word32 helloSz,
             now = TimeNowInMilliseconds();
             if (now == (word32)GETTIME_ERROR)
                 return now;
-            diff = now - ssl->session.ticketSeen;
+            if (now < ssl->session.ticketSeen)
+                diff = (0xFFFFFFFFU - ssl->session.ticketSeen) + now;
```

Comment:
Why is this needed?
This will put it off by 1 millisecond.

Reply:
If the time in milliseconds elapses 49 days then it will possibly rollover. This will make sure the new value is calculated correctly. I adjusted the off by 1 math error. Thanks for spotting that.

Reply:
Rollover in TLS 1.3 should be fine.
diff has some values added and subtracted to return it to 0 base.
The only time I can see this change making a difference is when 'int' is 64 bits.

Reply:
@SparkiDev , to me it looks like `ssl->session.ticketSeen` is set with a value from `TimeNowInMilliseconds()`, thus would need to handle the 49 day rollover case?